boards/apollo3/lora_things_plus/README.md

```diff
@@ -99,5 +109,25 @@ $ elf2tab -n ping-pong --stack 2048 --app-heap 1024 --kernel-heap 1024 --kernel-
 Then in the Tock repo you can flash the app with:
 
 ```shell
-$ make flash; APP=LoRaMac-node/build/src/apps/ping-pong//ping-pong.tbf make flash-app
+$ make flash; APP=LoRaMac-node/build/src/apps/ping-pong/ping-pong.tbf make flash-app
+``
+
+### RadioLib
+
+RadioLib is a Universal wireless communication library for embedded devices.
+
+There is currently a PR open for adding support to Tock as well:
+https://github.com/jgromes/RadioLib/pull/782
```

Comment:
It looks like the PR might be merged post-haste. Does it make sense to wait on that, update the README, then merge?

Reply:
I'm hoping it will be, but it's hard to know for sure. I'm happy to just update the README once it is merged

Reply:
Updated with the upstream status!